test-suite/build.gradle

```diff
@@ -25,8 +25,3 @@ dependencies {
 tasks.named('test') {
     useJUnitPlatform()
 }
-
-java {
```

Comment:
should we set: 

```
java {
    sourceCompatibility = JavaVersion.toVersion("17")
    targetCompatibility = JavaVersion.toVersion("17")
}
```

Create a test-base in buildScript to avoid repetition.

Reply:
I have created a PR https://github.com/micronaut-projects/micronaut-email/pull/135